shared/util/feature-flags.js

```diff
@@ -8,23 +8,28 @@ import getenv from 'getenv'
 const tracker2Key = 'tracker2'
 const loginKey = 'login'
 const mobileAppsExistKey = 'mobileAppsExist'
+const dz2Key = 'dz2'
 
 type FeatureFlags = {
   'tracker2': boolean,
   'login': boolean,
-  'mobileAppsExist': boolean
+  'mobileAppsExist': boolean,
+  'dz2': boolean
 }
 
 let features = getenv.array('KEYBASE_FEATURES', 'string', '')
 
-const tracker2 = features.includes(tracker2Key)
+// dz2 implies tracker2
```

Comment:
Is this okay? I'd like us to be able to turn on "dz2" (tracker+pinentry) with one feature flag so we can ship that change.